app/editor/src/features/admin/reports/ReportFormSections.tsx

```diff
@@ -44,7 +45,7 @@ export const ReportFormSections = () => {
       [...values.sections, defaultReportSection(values.id)].map((section, index) => {
         return {
           ...section,
-          name: `section-${index + 1}`,
+          name: `section-${index + 1}-${now.getTime()}`,
```

Comment:
I don't like this solution however it appears to be necessary due to how Postgres and EF update the sections which require a unique name.  The issue occurs when sections are moved around which would result in them receiving a new name based on their position.